chord_metadata_service/phenopackets/api_views.py

```diff
@@ -97,9 +89,6 @@ class BiosampleViewSet(ExtendedPhenopacketsModelViewSet):
     post:
     Create a new biosample
     """
-    queryset = m.Biosample.objects.all() \
-        .prefetch_related(*BIOSAMPLE_PREFETCH) \
-        .order_by("id")
```

Comment:
Why was this removed?

Reply:
not visible in the diff, but it's a duplicate - if you expand below you can see its there a second time